content/docs/user-guide/basic-concepts/workspace.md

```diff
@@ -6,3 +6,40 @@ tooltip: >-
   models, etc. Typically, it's also a Git repository. It will contain your DVC
   project.
 ---
+
+<!-- keywords: data science project architecture, machine learning project architecture, machine learning workflow, data science workflow, machine learning file system, data science file system, data science project structure, machine learning project structure, notebook version control -->
+
+# Workspace
+
+A data science project consists of data obtained from many different sources.
+This data may be split into multiple files or directories or (as the project
+structure needs) have different versions for different requirements. e.g. A
+smaller / simplified version might be required in prototyping for faster
+feedback and shorter training times. A single workspace to manage all artifacts
+of a project is desirable, although versioning needs and managing dependencies
+make it increasingly complex.
+
+DVC allows a single directory to contain all your project artifacts. In the
+documentation the workspace is the _user visible_ part of the directory that
+contains all your <abbr>project</abbr> files e.g. raw datasets, source code, ML
```

Comment:
```suggestion
DVC allows a single directory to contain all your project artifacts.
The workspace is the directory containing the _user visible_ part of
your <abbr>project</abbr> e.g. raw datasets, source code, ML
```

Reply:
Fixed in 50964b1 

Thank you 👍🏼

Reply:
Np. These copy edits can be easily committed from GH btw but as you prefer.